pkgs/tools/networking/twa/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, bash, gawk, curl, netcat }:
+
+stdenv.mkDerivation rec {
+  name = "twa-${version}";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "twa";
+    rev = "${version}";
+    sha256 = "16x9nzsrf10waqmjm423vx44820c6mls7gxc8azrdqnz18vdy1h4";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [ bash gawk curl netcat ];
+
+  installPhase = ''
+    install -Dm 0755 twa "$out/bin/twa"
```

Comment:
This needs to be wrapped with makeWrapper to have curl, netcat and ncurses in its PATH:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/phantomjs2/default.nix#L91